ping.go

```diff
@@ -309,12 +308,16 @@ func (p *Pinger) Run() {
 		if conn = p.listen(ipv4Proto[p.protocol]); conn == nil {
 			return
 		}
-		conn.IPv4PacketConn().SetControlMessage(ipv4.FlagTTL, true)
+		if conn.IPv4PacketConn().SetControlMessage(ipv4.FlagTTL, true) != nil {
+			return
```

Comment:
Would it not be better to capture and return these errors rather than swallowing them?

Reply:
This function doesn't return anything, so the best it can do is bail. :frowning: 

Changing `recvICMP()` to return an error is something else entirely, so IMO, out of scope for this change.

Reply:
What I meant is that modifying `func (p *Pinger) Run()` so that it returns an `error` isn't really a breaking change since the worst that can happen is peoples' linters complain about the ignored return value of `pinger.Run()`. But I take your point that maybe re-engineering how all errors are handled is something for a separate PR.

Reply:
https://github.com/go-ping/ping/pull/81 just needs a rebase. :grin: